var/spack/repos/builtin/packages/ispc/package.py

```diff
@@ -17,22 +17,27 @@ class Ispc(Package):
     homepage = "https://github.com/ispc/ispc/"
     url      = "https://netix.dl.sourceforge.net/project/ispcmirror/v1.10.0/ispc-v1.10.0-linux.tar.gz"
 
-    version('1.10.0', sha256='453211ade91c33826f4facb1336114831adbd35196d016e09d589a6ad8699aa3')
+    version(
+        '1.10.0',
+        sha256='453211ade91c33826f4facb1336114831adbd35196d016e09d589a6ad8699aa3',
+    )
```

Comment:
Why break this but not line 18?

Reply:
because of deeper indentation.. that's black just automating things..  either way, I don't mind. Someone with superpowers should please merge once it's good :)

Reply:
It minimizes diff of future changes

Reply:
With inline highlighting like in Github, I kind of doubt the benefit of doing this excessively.

Reply:
Also…

> ### NOTE: This is a beta product
> Black is already successfully used by several projects, small and big. It also sports a decent test suite. However, it is still very new. Things will probably be wonky for a while.

Sounds like yet another hipster bandwagon.

Reply:
See [contribution guide](https://spack.readthedocs.io/en/latest/contribution_guide.html?highlight=flake8#flake8-tests) to spack : 

```

It automatically adds approved exemptions from the flake8 checks.
For example, URLs are often longer than 80 characters, so we exempt
them from line length checks. We also exempt lines that start with
“homepage”, “url”, “version”, “variant”, “depends_on”, and “extends”
in package.py files.
```

On upstream reviews people suggest not to indent for above constructs...so I wouldn't do it here.